.github/workflows/Tests.yml

```diff
@@ -54,26 +54,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9, "3.10", "3.11", "3.12"]
+        python-version: ["3.10", "3.11", "3.12"]
         lang-var: ["de.utf-8", "jp.utf-8"]
         experimental: [true]
         include:
-          - python-version: 3.7
-            lang-var: "en_US.utf-8"
-            experimental: false
-          - python-version: 3.8
-            lang-var: "en_US.utf-8"
-            experimental: false
-          - python-version: 3.9
+          - python-version: "3.10"
             lang-var: "de.utf-8"
             experimental: false
-          - python-version: 3.9
+          - python-version: "3.10"
             lang-var: "jp.utf-8"
             experimental: false
-          - python-version: 3.9
+          - python-version: "3.10"
             lang-var: "en_US.utf-8"
             experimental: false
-          - python-version: "3.10"
+          - python-version: "3.11"
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to migrate to stronger support of 3.11 here